evap/evaluation/views.py

```diff
@@ -94,3 +102,26 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+@login_required
+def feedback_send(request):
+    sender_email = request.user.email
+    message = request.POST.get("message")
+    subject = "Feedback from {}".format(sender_email)
+
+    if message:
+        mail = EmailMessage(
+            subject=subject,
+            body=message,
+            to=[settings.FEEDBACK_EMAIL])
+
+        try:
+            mail.send()
+            logger.info('Sent feedback email: \n%s\n' % mail.message())
```

Comment:
the lint suggested to pass the formatting parameters to logger.info, not interpolate the string via %.